addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1466,6 +1466,7 @@ export class Wysiwyg extends Component {
                         data.rel = 'ugc';
                     }
                     data.linkDialog.applyLinkToDom(data);
+                    this.odooEditor.historyUnpauseSteps();
```

Comment:
This line is already in `close` which is going to be called before `onSave`. Therefore, I don't understand how this can fix the bug since the steps should already be unpaused at this point.